kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -553,18 +552,33 @@ retrycmd_if_failure 5 10 60 az account set --subscription $TENANT_SUBSCRIPTION_I
 #####################################################################################
 # aks-engine commands
 
-log_level -i "Generating ARM template using AKS-Engine."
-# No retry, generate does not call any external endpoint
-./bin/aks-engine generate $AZURESTACK_CONFIGURATION || exit $ERR_AKSE_GENERATE
 
-log_level -i "ARM template saved in directory $PWD/_output/$MASTER_DNS_PREFIX."
+log_level -i "Deploying using AKS Engine."
```

Comment:
Do we still need `az login`?

If not, we can also remove `add_azurecli_source`